deckhouse-controller/pkg/controller/module-controllers/release/controller.go

```diff
@@ -80,8 +83,19 @@ type Controller struct {
 	restartReason string
 }
 
+const (
+	approvalAnnotation     = "release.deckhouse.io/approved"
+	defaultScanInterval    = 15 * time.Second
```

Comment:
Let's rename this. For `defaultCheckInterval` or something like that

Reply:
done